packages/runner/lib/sdk/persist.ts

```diff
@@ -225,4 +233,32 @@ export class PersistClient {
         }
         return res;
     }
+
+    public async getRecords({
+        environmentId,
+        nangoConnectionId,
+        model,
+        cursor,
+        externalIds
+    }: {
+        environmentId: number;
+        nangoConnectionId: number;
+        model: string;
+        cursor?: string | undefined;
+        externalIds?: string[] | undefined;
+    }): Promise<Result<GetRecordsSuccess>> {
```

Comment:
There is no way to import and reuse type like we do with the public api?

Reply:
Not sure what you mean. I was just following the pattern for `getCursor` above.

Reply:
yeah I'm just pointing out we are not using shared types which could lead to discrepancy

Reply:
Ah yes I remember, always annoying this kind of decisions